addons/binding/org.openhab.binding.jeelink/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,224 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="jeelink" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- JeeLink USB Receiver Bridge Type -->
+	<bridge-type id="jeelink">
+		<label>@text/bridge-type.jeelink.label</label>
+		<description>@text/bridge-type.jeelink.description</description>
+
+		<config-description>
+			<parameter name="portName" type="text" required="true">
+				<label>@text/parameter.portname.label</label>
+				<description>@text/parameter.portname.description</description>
+				<default>serial:///dev/ttyUSB0</default>
+			</parameter>
+			<parameter name="sketchName" type="text" required="true">
+				<label>@text/parameter.sketchname.label</label>
+				<description>@text/parameter.sketchname.description</description>
+				<default>LaCrosseITPlusReader</default>
+
+				<options>
+					<option value="LaCrosseITPlusReader">LaCrosseITPlusReader</option>
+					<option value="ec3kSerial">ec3kSerial</option>
+				</options>
+			</parameter>
+            <parameter name="initCommands" type="text" required="false">
+                <label>@text/parameter.initcommands.label</label>
+                <description>@text/parameter.initcommands.description</description>
+            </parameter>
+		</config-description>
+
+	</bridge-type>
+
+	<!-- Lacrosse Temperature Sensor Thing Type -->
+	<thing-type id="lacrosse">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="jeelink" />
+		</supported-bridge-type-refs>
+
+        <label>@text/thing-type.lacrosse.label</label>
+        <description>@text/thing-type.lacrosse.description</description>
+
+		<channels>
+			<channel id="sensorId" typeId="sensor-id-channel" />
+			<channel id="sensorType" typeId="sensor-type-channel" />
+			<channel id="temperature" typeId="temperature-channel" />
+			<channel id="humidity" typeId="humidity-channel" />
+			<channel id="batteryNew" typeId="battery-new-channel" />
```

Comment:
It is not fake, it is an indication. The percent values are meant to provide a battery level for showing an according icon (which usually anyhow only has ~4 steps). There isn't a single battery powered device where such values are anywhere close to be precise.